usaspending_api/api_contracts/contracts/v2/search/spending_by_transaction_grouped.md

```diff
@@ -17,20 +17,25 @@ Searches for transaction records based on a provided set of filters and groups t
 
     + Attributes (object)
         + `filters` (required, AdvancedFilterObject)
-            These filters are applied at the Transaction level (`children`). They will not affect the Prime Awards (`results`) themselves. Must at least provide the following in the filters object, `keywords` and `award_type_codes`.
+            These filters are applied at the Transaction level. They will not affect the Prime Awards themselves. Must at least provide `award_type_codes` in the filters object.
         + `fields` (required, FieldNameObject)
```

Comment:
If we can remove `fields` then I would say remove this as well; and other usages following this in the contract